src/op_mode/firewall.py

```diff
@@ -90,6 +90,36 @@ def get_nftables_details(family, hook, priority):
         out[rule_id] = rule
     return out
 
+def get_nftables_state_details(family):
+    if family == 'ipv6':
+        suffix = 'ip6'
+        name_suffix = 'POLICY6'
+    elif family == 'ipv4':
+        suffix = 'ip'
+        name_suffix = 'POLICY'
+    else:
+        # no state policy for bridge
+        return {}
+
+    command = f'sudo nft list chain {suffix} vyos_filter VYOS_STATE_{name_suffix}'
```

Comment:
Please move the `sudo` call outside of this script and rather call the op-mode script with sudo in front from the XML part.

Reply:
Fixed, and also remove other `sudo` in the same file.